devops/docker/docker-compose.yml

```diff
@@ -11,6 +11,7 @@ services:
       - elasticsearch
     volumes:
       - ../../htdocs:/var/www/elastic-erga
+      - ../../components:/var/www/html/build/js/components
```

Comment:
Is it possible to change this path to be relative to /var/www/elastic-erga folder?